src/components/CourseInformation/CourseImage/CourseImage.js

```diff
@@ -1,20 +1,14 @@
-import classes from "../CourseInformation.module.css";
-
-const CourseImage = (props) => {
-  const img = props.img;
-
-  if (!img) {
+const CourseImage = ({ image }) => {
```

Comment:
I might be reading this wrong but it looks like the code checks if the course has an image and if not, it shows the gradient background. Shouldn't it be: course image then backend configuration image and then gradient?